frontend/src/pages/pipelines/global/experiments/artifacts/ArtifactUriLink.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Flex, FlexItem, Truncate } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { Artifact } from '~/third_party/mlmd';
+
+export const ArtifactUriLink: React.FC<{ artifact: Artifact.AsObject }> = ({ artifact }) => (
+  <Link to={artifact.uri} target="_blank">
```

Comment:
KF has this as a download link, when we have an s3 endpoint this opens a blank page instead of dowlaoding

Reply:
This has been addressed.